server/Arcadia.Assistant.Configuration/Configuration/SharepointSettings.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace Arcadia.Assistant.Configuration.Configuration
+{
+    using System.ComponentModel.DataAnnotations;
+
+    public class SharepointSettings : ISharepointSettings
+    {
+        [Required]
+        public string ServerUrl { get; set; }
+
+        [Required]
+        public string ClientId { get; set; }
+
+        [Required]
+        public string ClientSecret { get; set; }
+
+        public string CalendarEventIdMapping { get; set; }
```

Comment:
maybe CalendarEventIdField? With mapping I expect to see a pair here...

Reply:
Done.